jenkins.go

```diff
@@ -546,15 +509,18 @@ func (j *Jenkins) Poll() (int, error) {
 }
 
 // Creates a new Jenkins Instance
-// Optional parameters are: username, password
+// Optional parameters are: client, username, password
```

Comment:
Any need to briefly describe the usage of client like other third party packages?